docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md

```diff
@@ -0,0 +1,153 @@
+    | Required                | Whether the input is required.                                                                                                                                                                                                            |
+    | Default value           | Whether the variable has a default value.                                                                                                                                                                                                 |
+    | Mask value              | Whether the input is masked with asterisks in the layer YAML configuration. When you export a profile with masked variables, they will be masked in the exported profile. Upon import, you will be required to provide the masked values. |
+    | Hidden                  | Whether the variable is hidden during cluster deployment.                                                                                                                                                                                 |
+    | Read-only               | Whether the variable is editable during cluster deployment.                                                                                                                                                                               |
+
+    :::info
+
+    If a variable is required and hidden, it must contain a default value.
+
+    :::
+
+9.  Click **Create**.
+
+10. On the **Profile variables** pane, next to your variable, click the **Copy to clipboard** icon to copy the variable
+    name.
+
+11. Navigate to the YAML configuration of the profile layer for which you want to add the variable.
+
+12. Paste the variable in the `parameter: "{{.spectro.var.variable_name}}"` format and click **Confirm Updates**.
+
+    Alternatively, you can start typing the profile variable name with `{{.spectro.var.}}`, and Palette will suggest
+    profile variables that you can autocomplete.
+
+    ![Palette YAML editor with the added profile variables.](/profiles_create-cluster-profiles_define-profile-variables_add-vars-to-yaml.png)
+
+13. Repeat the steps described in this guide to define more variables and add them to the necessary cluster profile
+    layers.
+
+## Validate
+
+1. Log in to Palette.
+2. From the left **Main Menu**, select **Profiles** and navigate to the cluster profile for which you have defined
+   profile variables.
+3. In the upper-right corner, click **Variables** and, on the **Profile variables** pane, check that the necessary
+   variables are defined.
+4. Open the necessary profile layers and check that their YAML configuration contains the expected variables.
+
+## Manage Profile Variables
```

Comment:
This entire section feels a bit awkward. It feels like a how-to, but it doesn't follow our layout. Personally, I find it a bit strange to start with a warning message and that the section only contains bullet points.

We can make it an actual how-to section and add a prerequisite of having an existing cluster profile with variables, along with edit permission.

Reply:
Sorry about that, slipped away and didn't catch that I wasn't following the template! Fixed now